source/Printers.c

```diff
@@ -5,13 +5,17 @@
 // Constructing a new data type that represents HelloWorld's function pointer.
 typedef void(WINAPI* HelloWorldFunctionPointer)();
 
-void PrintMemoryError(LPCWSTR pCFPoint)
+void PrintMemoryError(
+	LPCWSTR pCFPoint
+)
 {
 	wprintf(L"[X] Failed To Allocate Memory For %s!\nExiting With Error Code : % x\n", pCFPoint, GetLastError());
 	return;
 }
 
-void PrintDrives(LPWSTR pPath) 
+void PrintDrives(
```

Comment:
actually this function is more generic and has nothing to do with drives, so you can rename it if you'd like:
```suggestion
void PrintWcharArray(
```